### DISCUSSION TITLE
Workflow Functionality Issue on the Dify Online Platform

### DISCUSSION THREAD

**Discussion by jailbreak07** [langgenius/dify#3926] [Help]
Workflows created on the Dify online platform can be tested normally during the orchestration stage. 
However, after being published (link: https://udify.app/workflow/GrFgTCTfZ4yghaQZ), the workflow fails to run properly and throws the following error:
"Application error: a client-side exception has occurred (see the browser console for more information)."

I found other publicly shared workflows on Twitter that exhibit the same error, indicating that this is likely not an individual issue. I would greatly appreciate if you could look into this matter. Thank you in advance for your assistance.
![Uploading screenshot-20240427-212704.png…]()

**Comment by prozaccc** [comment]
I've encountered the same issue as you.
